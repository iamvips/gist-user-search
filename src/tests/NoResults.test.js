import React from "react";
import renderer from "react-test-renderer";
import NoResults from "../components/NoResults";

it("should render correctly", () => {
  const tree = renderer.create(<NoResults />).toJSON();
  expect(tree).toMatchSnapshot();
});
