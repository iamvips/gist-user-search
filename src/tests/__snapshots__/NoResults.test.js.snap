// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="sc-bdvvtL leDNwJ"
  data-testid="no-results"
>
  <svg
    fill="currentColor"
    height="2.5em"
    stroke="currentColor"
    strokeWidth="0"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 16 16"
    width="2.5em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"
    />
  </svg>
  <h2>
    We couldnâ€™t find any matching gists.
  </h2>
</div>
`;
